src/main/java/net/dv8tion/jda/api/MessageBuilder.java

```diff
@@ -854,112 +833,6 @@ public MessageBuilder mentionRoles(@Nonnull long... roles)
-     * @deprecated This is not a reliable way to remove mentions from the content,
-     *             you should use {@link #setAllowedMentions(Collection)} instead.
-     */
-    @Nonnull
-    @Deprecated
-    @ForRemoval(deadline = "4.4.0")
-    @ReplaceWith("setAllowedMentions(Collections.emptyList())")
-    @DeprecatedSince("4.2.0")
-    public MessageBuilder stripMentions(@Nonnull Guild guild)
-    {
-        // Note: Users can rename to "everyone" or "here", so those
-        // should be stripped after the USER mention is stripped.
-        return this.stripMentions(guild.getJDA(), guild, Message.MentionType.values());
-    }
-
-    /**
-     * Removes all mentions of the specified types and replaces them with the closest looking textual representation.
-     *
-     * <p>Use this over {@link #stripMentions(JDA, Message.MentionType...)} if {@link net.dv8tion.jda.api.entities.User User} mentions should
-     * be replaced with their nicknames in a specific guild based.
-     * <br>Uses {@link net.dv8tion.jda.api.entities.Member#getEffectiveName()}
-     *
-     * @param  guild
-     *         the guild for {@link net.dv8tion.jda.api.entities.User User} mentions
-     * @param  types
-     *         the {@link net.dv8tion.jda.api.entities.Message.MentionType MentionTypes} that should be stripped
-     *
-     * @return The MessageBuilder instance. Useful for chaining.
-     *
-     * @deprecated This is not a reliable way to remove mentions from the content,
-     *             you should use {@link #setAllowedMentions(Collection)} instead.
-     */
-    @Nonnull
-    @Deprecated
-    @ForRemoval(deadline = "4.4.0")
-    @ReplaceWith("denyMentions(types)")
-    @DeprecatedSince("4.2.0")
-    public MessageBuilder stripMentions(@Nonnull Guild guild, @Nonnull Message.MentionType... types)
-    {
-        return this.stripMentions(guild.getJDA(), guild, types);
-    }
-
-    /**
-     * Removes all mentions of the specified types and replaces them with the closest looking textual representation.
-     *
-     * <p>Use this over {@link #stripMentions(Guild, Message.MentionType...)} if {@link net.dv8tion.jda.api.entities.User User}
-     * mentions should be replaced with their {@link net.dv8tion.jda.api.entities.User#getName()}.
-     *
-     * @param  jda
-     *         The JDA instance used to resolve the mentions.
-     * @param  types
-     *         the {@link net.dv8tion.jda.api.entities.Message.MentionType MentionTypes} that should be stripped
-     *
-     * @return The MessageBuilder instance. Useful for chaining.
-     *
-     * @deprecated This is not a reliable way to remove mentions from the content,
-     *             you should use {@link #setAllowedMentions(Collection)} instead.
-     */
-    @Nonnull
-    @Deprecated
-    @ForRemoval(deadline = "4.4.0")
-    @ReplaceWith("denyMentions(types)")
-    @DeprecatedSince("4.2.0")
-    public MessageBuilder stripMentions(@Nonnull JDA jda, @Nonnull Message.MentionType... types)
```

Comment:
I think there is a bit related code here that also needs to be removed like SplitPolicy and the private helper functions